iOS/DuckDuckGo/DataBrokerProtectionDebugViewController.swift

```diff
@@ -271,6 +390,185 @@ final class DataBrokerProtectionDebugViewController: UITableViewController {
+        alert.addAction(cancelAction)
+
+        present(alert, animated: true)
+    }
+
+    private func resetWebUICustomURL() {
+        webUISettings.setURLType(.production)
+        webUISettings.setCustomURL(webUISettings.productionURL)
+    }
+
+    private func useWebUIProductionURL() {
+        webUISettings.setURLType(.production)
+    }
 
+    private func useWebUICustomURL() {
+        webUISettings.setURLType(.custom)
+        webUISettings.setCustomURL(webUISettings.productionURL)
+    }
+
+    // MARK: - DBP API Actions
+
+    private func setCustomServiceRoot() {
+        let alert = UIAlertController(title: "Set Custom DBP API Service Root",
+                                      message: "Enter the base URL for the DBP API. This value is only applied when using the staging environment. Leave empty to reset to default.\n\n⚠️ Please reopen PIR and trigger a new scan for the changes to show up.",
+                                      preferredStyle: .alert)
+
+        alert.addTextField { textField in
+            textField.text = self.settings.serviceRoot
+        }
+
+        let saveAction = UIAlertAction(title: "Save", style: .default) { [weak self, weak alert] _ in
+            guard let textField = alert?.textFields?.first,
+                  let value = textField.text else {
+                return
+            }
+
+            self?.settings.serviceRoot = value
+            try? self?.manager.deleteAllData()
+            try? self?.forceBrokerJSONFilesUpdate()
+            self?.tableView.reloadData()
+        }
+
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+
+        alert.addAction(saveAction)
+        alert.addAction(cancelAction)
+
+        present(alert, animated: true)
+    }
+
+    private func removeAllBrokerData() {
+        do {
+            try manager.deleteAllData()
+            Logger.dataBrokerProtection.log("Successfully removed all broker data.")
+        } catch {
+            Logger.dataBrokerProtection.error("Failed to remove all broker data: \(error.localizedDescription)")
+        }
+    }
+    
+    // MARK: - Remote Broker JSON Service Usage
+    
+    private func checkForBrokerUpdates() {
+        guard let brokerUpdater = brokerUpdater else {
+            Logger.dataBrokerProtection.error("Failed to initialize RemoteBrokerJSONService")
+            return
+        }
+        
+        Task {
+            do {
+                try forceBrokerJSONFilesUpdate()
+                Logger.dataBrokerProtection.log("Successfully checked for broker updates")
+            } catch {
+                Logger.dataBrokerProtection.error("Failed to check for broker updates: \(error.localizedDescription)")
+            }
+        }
+    }
+
+    private func forceBrokerJSONFilesUpdate() throws {
+        Task {
+            settings.resetBrokerDeliveryData()
+            try await brokerUpdater?.checkForUpdates(skipsLimiter: true)
+        }
+    }
```

Comment:
I've added support for refreshing the remote broker JSON files. Note that this doesn't work out of the box on iOS since the feature flag is disabled, so I'm going to enable that for internal users so that we can test it during the beta. Hardcoding the flag to `true` and then triggering this does refresh the brokers correctly.